<launch>
  <master auto="start"/>
  <param name="/trex/start_dir" value="$(find executive_trex_pr2)/test/master/master.2"/>
  <include file="$(find executive_trex_pr2)/test/master/real.launch" />
</launch>
